### PR TITLE
Update docs for 0.9b1

### PR DESCRIPTION
**Pull request by minrk**
After this, I think we can cut the first 0.9b1 release, and make 0.7b1 of the helm chart.

### CODE REVIEW

File: kubespawner/spawner.py [hunk 1]
```diff
@@ -641,8 +642,8 @@ def _hub_connect_port_default(self):
            - `mountPath` The path on the container in which we want to mount the volume.
            - `name` The name of the volume we want to mount, as specified in the `volumes` config.
 
-        See `the Kubernetes documentation <https://kubernetes.io/docs/concepts/storage/volumes>`_ for more information on how
-        the `volumeMount` item works.
+        See `the Kubernetes documentation <https://kubernetes.io/docs/concepts/storage/volumes>`__
```
Comment (consideRatio) [inline kubespawner/spawner.py]: Another double underscore, I did not know about this syntax, I've seen a single one being used within the z2jh docs throughout.

File: docs/source/objects.rst [hunk 2]
```diff
@@ -2,13 +2,8 @@
 Objects
 =======
 
-Module: :mod:`kubespawner.objects`
-----------------------------------
-
 .. automodule:: kubespawner.objects
 
-.. currentmodule:: kubespawner.objects
-
```
Comment (consideRatio) [inline docs/source/objects.rst]: AHA! Have this perhaps caused documentation duplications in the past? I remember such bug.

**Comment by consideRatio** [comment]
LGTM! I'd really like to see #238 and #239 in the release though, while I also think we should wait with #245 as @betatim suggest.

File: kubespawner/spawner.py (hunk 1 above)
Comment (minrk) [reply kubespawner/spawner.py]: double-underscore means "anonymous link" whereas single underscore is "named link" which creates a named ref other links can refer to. Having multiple named refs with the text "Kubernetes documentation" was causing a collision.

File: docs/source/objects.rst (hunk 2 above)
Comment (minrk) [reply docs/source/objects.rst]: I think this was harmless, it's just duplicated in automodule as well.

File: kubespawner/spawner.py (hunk 1 above)
Comment (consideRatio) [reply kubespawner/spawner.py]: Ah!

**Comment by consideRatio** [comment]
Is this ready for merge?

**Pull request merged**